spec/factories/topic_section.rb

```diff
@@ -15,6 +15,7 @@
     topic
     title "Topic Section Title"
     description "Topic Section Description"
+    position 0
```

Comment:
Is this intentionally different to the acts_as_list default of 1? Maybe it doesn't matter.. A new section will have a position of `max + 1` or `0`.. even if the max is 0 it will still order correctly..